fsButler/utils.py

```diff
@@ -353,11 +376,12 @@ def matchMultiBand(butler, dataType, filters=['HSC-G', 'HSC-R', 'HSC-I', 'HSC-Z'
 
     for i in range(1, len(filters)):
         matched = strictMatch(matched, cats[i], multiMeas=multiMeas)
-    
+
     return matched
 
+
 def matchCats(cat1, cat2, matchRadius=1*afwGeom.arcseconds, includeMismatches=False,
-              multiMeas=False, suffix='.2'):
+              multiMeas=False, suffix='2'):
```

Comment:
The change to `suffix` seems kinda gratuitous. Is it necessary due to formatting of table field names?